tests/pipeline/test_us_equity_pricing_loader.py

```diff
@@ -175,43 +175,43 @@
     [
         # Before query range, should be excluded.
         {'declared_date': Timestamp('2015-05-01', tz='UTC').to_datetime64(),
-         'ex_date': Timestamp('2015-06-01', tz='UTC').to_datetime64(),
+         'ex_date': Timestamp('2015-06-02', tz='UTC').to_datetime64(),
```

Comment:
Edited above comment to switch 'dispersed' for 'checked for exclusion' when describing the date timing.